app/components/primer/alpha/action_bar_element.ts

```diff
@@ -86,16 +86,27 @@ class ActionBarElement extends HTMLElement {
       bindKeys: FocusKeys.ArrowHorizontal | FocusKeys.HomeAndEnd,
       focusOutBehavior: 'wrap',
       focusableElementFilter: element => {
-        return this.#isVisible(element)
+        return this.#checkVisibility(element)
       }
     })
   }
 
-  #isVisible(element: HTMLElement): boolean {
-    // Safari doesn't support `checkVisibility` yet.
-    if (typeof element.checkVisibility === 'function') return element.checkVisibility()
-
-    return Boolean(element.offsetParent || element.offsetWidth || element.offsetHeight)
+  #checkVisibility(el: HTMLElement, {checkOpacity = false, checkVisibilityCSS = false} = {}): boolean {
+    if (!el) return false
+    if (typeof el.checkVisibility === 'function') {
```

Comment:
@keithamus Is there some typescript config that needs updating for these errors?

<img width="948" alt="image" src="https://github.com/primer/view_components/assets/54012/c06ee15b-c30e-40ee-b6f7-023255c2b3a1">